### DISCUSSION TITLE
Incomplete Documentation for doc.tensor

### DISCUSSION THREAD

**Discussion by FloGro3** [explosion/spaCy#10803] [Help: Other Questions] [answered]
![image](https://user-images.githubusercontent.com/93530403/168346604-3aaaa30b-4805-4a49-8c9d-7146a4112c6f.png)
When using the pipeline "de_core_news_sm" from Spacy and running it with a GPU the type of doc.tensor isn't numpy.ndarray as stated in the documentation [Spacy Doc Attributes](https://spacy.io/api/doc).
As you can see the type is "cupy._core.core.ndarray" which means you can't use all the methods you could use with a numpy.ndarray. You'll get a numpy.ndarray when you perform a get() call an the tensor:
![image](https://user-images.githubusercontent.com/93530403/168347567-bac1dca5-4d8d-475b-881d-c3ae18b0af28.png)

Here's the code to reproduce the situation:

```
import spacy
spacy.require_gpu()
nlp = spacy.load("de_core_news_sm")
doc = nlp("Dies ist ein Test.")
type(doc.tensor)
type(doc.tensor.get())
```

A small tip or hint regarding this special situation would be nice in the documentation in my opinion :)

**Comment by rmitsch** [comment, marked as answer]
You're right, the docs could be clearer here. We'll update them. Thank for reporting this!
